prisma/schema.prisma

```diff
@@ -14,17 +14,51 @@ datasource db {
 }
 
 model Post {
-    id        Int      @id @default(autoincrement())
-    name      String
+    id              Int        @id @default(autoincrement())
+    name            String // event title
+    description     String     @db.Text
+    dateTime        DateTime
+    image           String?
+    price           Float      @default(0)
+    maxParticipants Int?       @default(0)
+    invitLink       String?
+    address         String
```

Comment:
I believe we were also going to be allowing users to host virtual events? Wouldn't this along with `lat` and `lng` then be nullable? We can treat null as a virtual event in the front end.